### PR TITLE
:robot: Add rpcview.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/691 :rocket:

### CODE REVIEW

File: packages/rpcview.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'RpcView'
```
Comment (binjo) [inline packages/rpcview.vm/tools/chocolateyinstall.ps1:4]:
```ps1
$toolName = 'RpcView64'
$executableName = 'RpcView.exe'
```

File: packages/rpcview.vm/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+$category = 'Utilities'
+
+$zipUrl = 'https://github.com/silverf0x/RpcView/releases/download/v0.3.1.90/RpcView64.7z'
+$zipSha256 = 'a1d89c9d81a2e9c7558e8f0c91ec8652d40af94726f3125f9fe31206adb528de'
+
+VM-Install-From-Zip $toolName $category $zipUrl -zipSha256 $zipSha256 -consoleApp $false
```
Comment (binjo) [inline packages/rpcview.vm/tools/chocolateyinstall.ps1:10]: As the folder name is different from the executable name, extra params needed.
```ps1
VM-Install-From-Zip $toolName $category $zipUrl -zipSha256 $zipSha256 -consoleApp $false -executableName $executableName
```

File: packages/rpcview.vm/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (binjo) [reply packages/rpcview.vm/tools/chocolateyinstall.ps1:4]: `-innerFolder $true` looks better, with the directory structure:
```
Extracting C:\Users\runneradmin\AppData\Local\Temp\chocolatey\rpcview.vm\0.3.1.20231018\RpcView64.7z to C:\Tools\RpcView...
C:\Tools\RpcView
Added C:\ProgramData\chocolatey\bin\RpcView.exe shim pointed to 'c:\tools\rpcview\rpcview64\rpcview.exe'.
C:\Tools\RpcView\RpcView64\RpcView.exe
 The upgrade of rpcview.vm was successful.
  Software installed to 'C:\Tools\RpcView'
```

File: packages/rpcview.vm/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'RpcView'
+$category = 'Utilities'
+# $executableName = 'RpcView.exe'
```
Comment (Ana06) [inline packages/rpcview.vm/tools/chocolateyinstall.ps1:6]: I think this needs to be removed 👀

**Review by Ana06 — COMMENTED** [review]
Thanks for the fix @binjo! There is one comment that needs to be removed.

Could you also squash the commits (in order to maintain a clean commit history? Alternatively, we can merge this PR with `squash and merge`

**Comment by binjo** [comment]
Looks I don't have permission to `squash and merge` online.

File: packages/rpcview.vm/rpcview.vm.nuspec [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>rpcview.vm</id>
+    <version>0.3.1.20231018</version>
+    <authors>silverf0x</authors>
+    <description>RpcView is an open-source tool to explore and decompile all RPC functionalities present on a Microsoft system</description>
+    <dependencies>
+      <dependency id="common.vm" />
+    </dependencies>
```
Comment (emtuls) [inline packages/rpcview.vm/rpcview.vm.nuspec:10]: This package requires "Microsoft Visual C++ 2019 Redistributable" in order to work. Adding this line under `<dependency id="common.vm" />` will make it work:

`<dependency id="vcredist140" version="14.36.32532"/>`

**Review by emtuls — CHANGES_REQUESTED** [review]
Missing dependency "Microsoft Visual C++ 2019 Redistributable" which is required for tool to work.

File: packages/rpcview.vm/rpcview.vm.nuspec [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>rpcview.vm</id>
+    <version>0.3.1.20231018</version>
+    <authors>silverf0x</authors>
+    <description>RpcView is an open-source tool to explore and decompile all RPC functionalities present on a Microsoft system</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="vcredist140" version="14.36.32532"/>
```
Comment (Ana06) [inline packages/rpcview.vm/rpcview.vm.nuspec:10]:
```suggestion
      <dependency id="vcredist140" version="[14.36.32532]" />
```

We should ping the dependency to ensure we test updates or limit it to a lower range like `version="[14.36.32532, 14.37)"`. This dependency is used also in [nmap](https://github.com/mandiant/VM-Packages/blob/a21b189ce92c24c1ca8a85e5d5692fdfccca0cb0/packages/nmap.vm/nmap.vm.nuspec#L12). I think we should create a metapackage to ensure both use the same version and avoid conflicts. I have added this to https://github.com/mandiant/VM-Packages/issues/673 (we can address it in a different PR).

**Review by Ana06 — CHANGES_REQUESTED** [review]
Thanks for all the work @binjo (and @emtuls for the review)

**Review by Ana06 — APPROVED** [review]

**Review by binjo — APPROVED** [review]

**Pull request merged**